test/KnoBlockIO.ts

```diff
@@ -1,260 +1,260 @@
 import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
-import { MockKnoBlockIO, MockKnoBlockIO__factory } from '../typechain-types';
+import { IKnoBlock } from '../typechain-types';
 import { ethers } from 'hardhat';
 import { BigNumber, ContractTransaction } from 'ethers';
 import { expect } from 'chai';
 
-describe('KnoBlockIO contract', function () {
-  describe('KnoBlockIO', function () {
-    let deployer: SignerWithAddress;
-    let addr1: SignerWithAddress;
-    let addr2: SignerWithAddress;
-    let instance: MockKnoBlockIO;
+export interface KnoBlockIOBehaviorArgs {}
 
-    const zero = ethers.constants.Zero;
-    const one = ethers.constants.One;
+export function describeBehaviorOfKnoBlockIO(deploy: () => Promise<IKnoBlock>) {
```

Comment:
Avoid magic numbers, replace them with constants instead.

If there are duplicated blocks of code, use helper functions (with explanative comments on what they do) declared at the top of the file or in a separate `helpers.ts` file.